src/components/data-model-table/table-wrapper.tsx

```diff
@@ -217,7 +204,6 @@ export const TableWrapper = forwardRef(function TableWrapper<
 
   const handleSearchInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     const { value } = e.target;
-    setSearchTerm(value);
```

Comment:
Remove this line makes the subscribe to search value (new line 195) to work, i.e., instead of pressing enter, table returns filtered result automatically after the debounceTime.

Reply:
good stuff! 🙌